tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -1249,3 +1249,31 @@ if (!/\/2\.7/.test(Cypress.env('rancher_version')) && !/\/2\.8/.test(Cypress.env
     )
   })
 };
+
+if (!/\/2\.7/.test(Cypress.env('rancher_version')) && !/\/2\.8/.test(Cypress.env('rancher_version'))) {
+  describe('Test Fleet Resource Count', { tags: '@p1'}, () => {
+    qase(155,
+      it("Fleet-155: Test 'clusters' incorrect resource count ", { tags: '@fleet-155' }, () => {
```

Comment:
Suggestion: Test clusters resource count is correct

Reply:
I will update it, thanks :smile:

Reply:
Done.